airflow/providers/apache/impala/provider.yaml

```diff
@@ -28,7 +28,7 @@ versions:
   - 1.0.0
 
 dependencies:
-  - impyla>=0.18.0,<1.0
+  - impyla[kerberos]>=0.18.0,<1.0
```

Comment:
Do we need this here? Can users not install this additionally in their setup if they need it? I mean for users who do not want to use Kerberos authentication and use other mechanisms like LDAP, for them we would be installing this additional dependency which many not be needed, no?

Reply:
Yes, indeed, this will bother users that do not need it. My reasoning was the following: For Hadoop, [the provider is shipped with kerberos](https://github.com/apache/airflow/blob/main/airflow/providers/apache/hdfs/provider.yaml#L46). Thus, to stay consistent, for Impala (which is setup on top of an hadoop system), it makes sense to have it bundled too. 

I can propose a PR to add kerberos as optional dependency to hdfs and impala ?

Reply:
okay. @eladkal could you please help us here on what could be the appropriate way?

Reply:
Edited the PR to make it optional. I'll open a PR to update the docs asap.